cli/src/semgrep/commands/ci.py

```diff
@@ -158,6 +158,13 @@ def fix_head_if_github_action(metadata: GitMeta) -> None:
 )
 @click.option("--code", is_flag=True, hidden=True)
 @click.option("--beta-testing-secrets", is_flag=True, hidden=True)
+@click.option(
+    "--secrets",
+    "run_secrets_flag",
+    is_flag=True,
+    hidden=True,
+    help="Enable support for secret validation. Requires Semgrep Secrets, contact support@semgrep.com for more informationon this.",
```

Comment:
nit: typo `informationon`

Reply:
feel free to ignore and resolve this comment though! just chiming in

Reply:
Thanks, I am fine either way. I just had already thought of what the help message should say, so whenever we don't want it hidden it will save someone a few cycles. I am going to keep if for the time being.